HopsanCore/include/HopsanCoreGitVersion.h

```diff
@@ -1,7 +1,7 @@
-// This file has been automatically generated by ./writeGitVersionHeader.sh
+// This file has been automatically generated by /home/robbr48/git/hopsan/HopsanCore/../writeGitVersionHeader.sh
```

Comment:
Do not commit this file, I know it inconvenient but I don't have a better solution right now